### PR TITLE
chore: adding Minh to codeowners

### PR DESCRIPTION
**Pull request by esacteksab**
# SC-870

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Default owners for everything
 
-* @abbyoung @esacteksab @gidjin
+* @abbyoung @esacteksab @gidjin @minhlai
 
 # Github settings
 
-.github/settings.yml @abbyoung @esacteksab @noahfirth @gidjin
+.github/settings.yml @abbyoung @esacteksab @noahfirth @gidjin @minhlai
```
Comment (gidjin) [inline .github/CODEOWNERS:7]: Suggestion(non-blocking): Drop Noah from owning the settings file

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 # CODEOWNERS prevent renovate automerge since it will automatically ask for review
 
 # See documentation for more details: https://docs.renovatebot.com/automerge-configuration/
 
 # Solution: make files used by renovate owned by everyone
 
-**/*/package.json
-**/*/yarn.lock
+**/\*/package.json
+**/\*/yarn.lock
```
Comment (gidjin) [inline .github/CODEOWNERS:16]: Question: was this an intentional change? Kind of looks like automated formatting going too far. Github seems to at least be flagging it as invalid format if it is intentional.

**Review by gidjin — COMMENTED** [review]
Looks good, though I'm wondering if we need to fix the last 2 lines before approving / merging this

File: .github/CODEOWNERS (hunk 2 above)
Comment (esacteksab) [reply .github/CODEOWNERS]: Nope. Silly linter. Thanks for the catch.

File: .github/CODEOWNERS (hunk 1 above)
Comment (esacteksab) [reply .github/CODEOWNERS]: Ooops! Thanks!

**Review by gidjin — APPROVED** [review]
lgtm

**Pull request merged**